src/app/content/components/Page/PageContent.tsx

```diff
@@ -60,6 +65,21 @@ export default styled(MainContent)`
         border-bottom: 1px solid ${style.focused};
       }
 
+      &.first.text {
+        &:before {
+          position: absolute;
+          top: -0.1rem;
+          left: -${highlightIndicatorSize - 0.3}rem;
+          content: "";
+          width: 0;
+          height: 0;
+          border-left: ${highlightIndicatorSize}rem solid transparent;
+          border-right: ${highlightIndicatorSize}rem solid transparent;
+          border-bottom: ${highlightIndicatorSize}rem solid ${style.focused};
+          transform: rotate(-45deg);
```

Comment:
I played around with this and was able to recreate it with 
```
position: absolute;
top: 0rem;
left: 0rem;
content: "";
width: 0;
height: 0;
border-left: 1.1rem solid #fed200;
border-top: 1.1rem solid transparent;
transform: rotate(90deg);
``` 
I think `0` for `top` and `left` will be a bit more reliable